adapter/comm/tests/test_tools.py

```diff
@@ -181,3 +185,13 @@ def test_compare_output2(self):
             dir2,
             "/Volumes/Abc/First_Level/Second_Level/Third_Level/input",
         )
+
+    def test_mark_time(self):
+        self.assertEqual(
+            len(mark_time(ts_format="short")), len("_220808_1051")
+        )
+        self.assertEqual(
+            len(mark_time(ts_format="long")), len("_2022_08_08-10h_51m")
```

Comment:
I wonder if we could just always use underscore ("_") in the time stamp like "_2022_08_08_10h_51m" rather than a combination of dash ("-") and underscore ("_") like "_2022_08_08-10h_51m". It's not a big change I think, but the former makes it easier to highlight and copy/paste paths.

Reply:
The long format is for users who want to use the current file name convention, it provides somewhat backward compatibility.

Reply:
I think it would be fine to replace the dash with the underscore as Lyra indicated, if it makes things easier.